androidApp/src/main/java/fr/paug/androidmakers/MainActivity.kt

```diff
@@ -72,9 +67,12 @@ class MainActivity : AppCompatActivity() {
             }
 
             MainLayout(
-              user = userState.value,
```

Comment:
Reason I removed it here is that it doesn't play well with Precompose (see [here](https://github.com/Tlaster/PreCompose/issues/271#issuecomment-2049144780))